src/lib/components/DeckGLMap/configuration.js

```diff
@@ -0,0 +1,50 @@
+// This configuration object determines which deck.gl classes are accessible in the serialized json
+
+import {
+    MapView,
+    FirstPersonView,
+    OrbitView,
+    OrthographicView,
+} from "@deck.gl/core";
+import * as Layers from "@deck.gl/layers";
+import * as AggregationLayers from "@deck.gl/aggregation-layers";
+import * as GeoLayers from "@deck.gl/geo-layers";
+import * as MeshLayers from "@deck.gl/mesh-layers";
+import * as CustomLayers from "./layers";
+
+import { COORDINATE_SYSTEM } from "@deck.gl/core";
```

Comment:
Just a very minor thing, the vs code has a short cut to organize import, (it is Shift+Alt+O on Windows, not sure about Mac...)

Reply:
I've just tried that, but it organizes everything alphabetically, combining imports from different libraries.
Now I've manually arranged them in alphabetical order, while keeping the imports grouped: React stuff, then DeckGL stuff, then other libraries, then local stuff. For me it's easier to visually "parse" them, but for consistency I can also use that shortcut. Let me know what you think :)

Reply:
Combining imports from different libraries sounds like an odd behavoir, I was expecting it would organise the imports by merging the ones from the same library. But yeah I think this doesnt matter. :)